pkg/vf/rosetta_amd64.go

```diff
@@ -0,0 +1,15 @@
+package vf
+
+import (
+	"fmt"
+
+	"github.com/Code-Hex/vz/v3"
+)
+
+func (dev *RosettaShare) toVz() (vz.DirectorySharingDeviceConfiguration, error) {
+	return nil, fmt.Errorf("rosetta is unsupported on non-arm64 platforms")
```

Comment:
will this prevent startup or only provide an error message?

Reply:
It will prevent startup (though I haven't tested it)